tests/test_38_organizations.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+include('test_utils.php');
+
+if (!class_exists('bigml')) {
+   include '../bigml/bigml.php';
+}
+
+use BigML\BigML;
+use BigML\BigMLRequest;
+
+class BigMLTestOrganizations extends PHPUnit_Framework_TestCase
+{
+    protected static $api;
+    protected static $api_org;
+    protected static $data_localfile = './data/iris.csv';
+    protected static $project;
+    protected static $test_name;
+
+    public static function setUpBeforeClass() {
+
+       $org = getenv("BIGML_TEST_ORGANIZATION");
```

Comment:
I'd change the name of the variable to BIGML_ORGANIZATION, to be consisten with naming in the rest of bindings and the dashboard.